test/users/engin/compiler/parentheseslessRecConst.chpl

```diff
@@ -0,0 +1,20 @@
+record independentRecord { var field = 5;} 
+
+var ir = new independentRecord;
+writeln(ir.field);
+//above lines compile with/without --verify
+
+class outerClass {
+  // var r: innerRecord; //doesn't work
+
+  // following line throws an internal bug error when compiled with --verify
+  // when compiled with --devel on top of that error message is more useful
```

Comment:
I'd make the sentence breaks more explicit in this comment, I initially read them as a single sentence and got confused.  The second sentence would read better if it were with a comma separating "that" and "error message", and probably using "the error message" instead of just "error message".